tsc/src/video/img_set.cpp

```diff
@@ -0,0 +1,406 @@
+/***************************************************************************
+ * img_set.cpp  -  Image set container
+ *
+ * Copyright © 2003 - 2011 Florian Richter
+ * Copyright © 2013 - 2014 The TSC Contributors
```

Comment:
This file is new, so the copyright notice should rather be

~~~~~~
Copyright © 2015 The TSC Contributors
~~~~~~

And nothing else. The README copyright notice is the “summary” notice of all copyright notices available in the game.

Vale,
Quintus